packages/devtools_app/lib/src/screens/memory/panes/chart/controller/chart_pane_controller.dart

```diff
@@ -34,13 +34,22 @@ class MemoryChartPaneController extends DisposableController
     );
   }
 
+  Map<String, dynamic> toJson() {
```

Comment:
nit: class member order: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#order-other-class-members-in-a-way-that-makes-sense

Reply:
They say 'If no particular order is obvious'.

fromJson and toJson are better readable when close to each other.